rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -58,6 +69,41 @@ public void resetConfigToRegTest() {
         when(config.getActivationConfig()).thenReturn(activationConfig);
     }
 
+    @Test
+    public void getActivePowpegRedeemScriptBeforeFork() throws VMException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(false);
+        doReturn(false).when(activationConfig).isActive(eq(RSKIP293), anyLong());
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withBridgeConstants(bridgeConstants)
+                .withActivations(activations)
+                .build();
+        Bridge bridge = getBridgeInstance(bridgeSupport);
+
+        byte[] data = BridgeMethods.GET_ACTIVE_POWPEG_REDEEM_SCRIPT.getFunction().encode(new Object[]{});
+
+        assertNull(bridge.execute(data));
+    }
+
+    @Test
+    public void getActivePowpegRedeemScriptAfterFork() throws VMException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(true);
+        doReturn(true).when(activationConfig).isActive(eq(RSKIP293), anyLong());
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withBridgeConstants(bridgeConstants)
+                .withActivations(activations)
+                .build();
+
+        Bridge bridge = getBridgeInstance(bridgeSupport);
+
+        byte[] data = BridgeMethods.GET_ACTIVE_POWPEG_REDEEM_SCRIPT.getFunction().encode(new Object[]{});
+
+        assertNotEquals(0, bridge.execute(data).length);
```

Comment:
I can do that, but I'm concerned about where should I get the redeem Script that will be used to compare with the result?  Should I get it from a real bridgeSupport instance like this?:

```
        BridgeSupport bridgeSupport = bridgeSupportBuilder
                .withBridgeConstants(bridgeConstants)
                .withActivations(activations)
                .build();
        Script redeemScript = bridgeSupport.getActiveFederation().getRedeemScript();
```

Asserting expected value vs result:

```
byte[] data = BridgeMethods.GET_ACTIVE_POWPEG_REDEEM_SCRIPT.getFunction().encode(new Object[]{});
        byte[] result = bridge.execute(data);

        assertTrue(
                Arrays.equals(
                        redeemScript.getProgram(),
                        (byte[]) BridgeMethods.GET_ACTIVE_POWPEG_REDEEM_SCRIPT.getFunction().decodeResult(result)[0]
                )
        );
```

Reply:
What I'm not sure is how much value this adds. I mean, I'm using the same script instance for mocking the method, then from this same script instance, I'm getting the result byte array to compare with the expected value.

Reply:
Since you are mocking the result of the BridgeSupport method you could basically return any Script value. An option could be to use the genesis federation redeem script

```
when(bridgeSupportMock.getActivePowpegRedeemScript()).thenReturn(BridgeRegTestConstants.getInstance().getRedeemScript())
```